### ISSUE TITLE
Publishing on BlackBerry World

### ISSUE THREAD

**Issue by cosmomill** [microg/android_packages_apps_GmsCore#105]
I would like to kindly request permission to publish the microG Services Core app v0.2.1 on BlackBerry World as far as the app will be accepted by RIM.

**Comment by mar-v-in** [comment]
I am certain that this app does not work properly on stock Blackberry systems with apps that use play services due to the signature verification. Did you verify that it actually runs and is useful on stock BlackBerry devices?

**Comment by cosmomill** [comment]
Yes, I can confirm that the microG Services Core app runs flawlessly on stock Blackberry systems as well as your BlankStore app. Of course, it's not possible to use a custom ROM with signature spoofing support or install the Xposed framework. To run apps which depend on play services on stock Blackberry systems the app must be patched. A very active CrackBerry user has developed the [Play Services Patcher](http://forums.crackberry.com/android-apps-amazon-store-apk-files-f413/how-patch-android-apps-depending-google-maps-google-play-services-914930/) to easily remove signature verification from apps. It is mainly for his own [pre-patched Google Play Services app](http://forums.crackberry.com/android-apps-amazon-store-apk-files-f413/cobalts-official-google-apps-landing-page-965257/) but woks also for microG Services Core. It seems there is a quite large community on CrackBerry which is interested in getting play services dependent apps running. I use the microG Services Core app to get voice calls working in Signal.
